Bottom: ad8dff35d50db2013dbb6cb5a132c844382e3732
Top:    3ec3e63ea069f3130c9bff5c9cad4da8d06401df
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 12:52:49 +0100

Refresh of add-strip-hierarchy-options-to

---

diff --git a/src/action.c b/src/action.c
index bada093..cf62b49 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4847,7 +4847,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
   char *function = ARG (0);
   if (function && PCB->ElementOn)
     {
-      HideCrosshair (true);
       switch (GetFunctionID (function))
 	{
 	case F_Object:
@@ -4863,7 +4862,7 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 		EraseElementName ((ElementTypePtr) ptr2);
 		TOGGLE_FLAG (HIDENAMEFLAG, (ElementTypePtr) ptr2);
 /* TODO: 		   SetTextBoundingBox (&PCB->Font, new); */
-		DrawElementName ((ElementTypePtr) ptr2, 0);
+		DrawElementName ((ElementTypePtr) ptr2);
 		Draw ();
 		IncrementUndoSerialNumber ();
 	      }
@@ -4885,7 +4884,7 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 		  EraseElementName (element);
 		  TOGGLE_FLAG (STRIPHIERFLAG, element);
 /* TODO: 		   SetTextBoundingBox (&PCB->Font, new); */
-		  DrawElementName (element, 0);
+		  DrawElementName (element);
 		  changed = true;
 		}
 	    }
@@ -4897,7 +4896,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 	      }
 	  }
 	}
-      RestoreCrosshair (true);
     }
   return 0;
 }
